docs/configurations/proxy.md

```diff
@@ -1,120 +1,225 @@
 # Proxy
```

Comment:
Probably it's just me but I needed to read this several times to understand how to use it and I wonder if we can simplify this doc a little bit.

Reply:
Sure, will look at the documentation and try to simplify it.